examples/osm-vector-tiles.js

```diff
@@ -10,6 +10,7 @@ goog.require('ol.style.Style');
 goog.require('ol.tilegrid');
 
 
+var key = 'vector-tiles-5eJz6JX';
```

Comment:
This will be cloaked in the example source code shown below the example, and when used in a JSFiddle.